tools/CMakeLists.txt

```diff
@@ -6,7 +6,13 @@ endfunction()
 
 function(pico_generate_pio_header TARGET PIO)
     _pico_init_pioasm()
-    cmake_parse_arguments(pico_generate_pio_header "" "OUTPUT_DIR" "" ${ARGN} )
+    cmake_parse_arguments(pico_generate_pio_header "" "OUTPUT_FORMAT;OUTPUT_DIR" "" ${ARGN} )
+
+    if (usdk_generate_pio_header_OUTPUT_FORMAT)
```

Comment:
The variable name should be something like `PIOASM_DEFAULT_OUTPUT_FORMAT`

Reply:
I just noticed a typo in there: it says "usdk_generate_pio_header_OUTPUT_FORMAT", which is incorrect, should be "pico_...". That variable is set by the cmake_parse function above and is just a temporary.

IMO it could make sense to set the output format to PIOASM_DEFAULT_OUTPUT_FORMAT if the OUTPUT_FORMAT option isn't set (as opposed to hardcoding c-sdk) as an additional way to customize it.

I'll fixup the patch series to fix the typo, and I can also add the PIOASM_DEFAULT_OUTPUT_FORMAT variable if you want.

Reply:
typo fixed in a8ce4cd  
default variable added in 546b89d